LondonBritishLibrary/orient/BLorient753.xml

```diff
@@ -133,13 +81,244 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                    <q xml:lang="gez">ቀሌምንጦስ፡  ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+                                </item>
+                            </list>
+                            <list>
+                              <item xml:id="e1">
+                                <desc>As appears from <locus target="#84r"></locus> the first owner was
+                              <persName ref="PRS13022Awesegneyos" role="owner"></persName>.</desc>
+                              </item>
+                            </list>
+                        </additions>
+                    </physDesc>
+
+                    </msPart>
+                    <msPart xml:id="p2">
+                        <msIdentifier><idno></idno></msIdentifier>
+                        <msContents>
+                          <msItem xml:id="p2_i1">
+                            <locus target="#156"></locus>
+                            <title type="incomplete" xml:lang="en">Conclusion of a
+                            unidentified paraenetic discourse</title>
+                          </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Leaf">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">1</measure>
+                                    </extent>
+                                </supportDesc>
+                                <layoutDesc>
+                                    <layout columns="3">
+                                    </layout>
+                                </layoutDesc>
+                            </objectDesc>
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h2">
+                                    <date notBefore="1600" notAfter="1800" resp="PRS10171Wright"/>
+                                </handNote>
+                            </handDesc>
+                        </physDesc>
+                    </msPart>
+                    <msPart xml:id="p3">
+                        <msIdentifier><idno></idno></msIdentifier>
+                        <msContents>
+                          <msItem xml:id="p3_i1">
+                            <locus target="#157"></locus>
+                            <title type="incomplete" xml:lang="en" ref="LIT1932Mashaf">
+                              <certainty locus="value" match="../@ref" cert="low"></certainty>
+                              Excerp from a Lectionary (New Testament)</title>
+                          </msItem>
```

Comment:
since you add xml lang en to the title it would be even more important to add an xml lang gez element for the text to avoid confusions